docs/versions.md

```diff
@@ -0,0 +1,65 @@
+# .NET Standard Versions
+
+The table below lists all versions of .NET Standard:
+
+* The columns represent .NET Standard versions. The header is a link to a
+  document that shows which APIs got added in that version of .NET Standard.
+* The rows indicate which version of a given .NET platform implemens a given
+  .NET Standard version.
+* The arrows indicate that the platform supports a higher version of .NET
+  Standard. For instance, .NET Core 1.0 supports the .NET Standard version 1.6,
+  which is why there are arrows pointing to the right for the lower versions 1.0
+  - 1.5.
+
+You can use this table to understand what the highest version of .NET Standard
+is that you can target, based on which .NET platforms you intend to run on. For
+instance, if you want to run on .NET Framework 4.5 and .NET Core 1.0, you can at
+most target .NET Standard 1.1.
+
+|.NET Platform              | [1.0]| [1.1]| [1.2]| [1.3]| [1.4]| [1.5]| [1.6]| [2.0]|
+|:--------------------------|-----:|-----:|-----:|-----:|-----:|-----:|-----:|-----:|
+|.NET Core                  |&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|   1.0| vNext|
+|.NET Framework             |&rarr;|  4.5 | 4.5.1|   4.6|&rarr;|&rarr;|&rarr;| 4.6.1|
+|Mono                       |&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;| vNext|
+|Xamarin.iOS                |&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;| vNext|
+|Xamarin.Android            |&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;| vNext|
+|Universal Windows Platform |&rarr;|&rarr;|&rarr;|&rarr;|  10.0|&rarr;|&rarr;| vNext|
+|Windows                    |&rarr;|   8.0|   8.1|      |      |      |      |      |
+|Windows Phone              |&rarr;|&rarr;|   8.1|      |      |      |      |      |
+|Windows Phone Silverlight  |   8.0|      |      |      |      |      |      |      |
```

Comment:
Nevermind, the changes in https://github.com/dotnet/standard/pull/61 explain that this is no longer true. Though that means that the "Introducing .NET Standard" blog post linked at the start of the FAQ is now out of date.